cip/1.accepted/CIP2016-12-19-Reserved-keywords.adoc

```diff
@@ -53,40 +53,46 @@ By _escaping_ any of the reserved words (encapsulating in backticks ```), they w
 
 ==== Reserved words
 
-We list the reserved words in the three categories from which they are drawn: keywords, operators, and literals.
+We list the reserved words grouped by the categories from which they are drawn.
 In addition to this, we list a number of words that are reserved for future use.
 
-===== Keywords
+===== Clauses
 
-* ALL
-* ASC
-* ASCENDING
-* BY
 * CREATE
-* CASE
 * DELETE
-* DESC
-* DESCENDING
 * DETACH
-* ELSE
-* END
 * EXISTS
-* LIMIT
 * MATCH
 * MERGE
-* ON
 * OPTIONAL
-* ORDER
 * REMOVE
 * RETURN
 * SET
+* UNION
+* UNWIND
+* WITH
+
+===== Subclauses
```

Comment:
I think that the following keywords:

`ASC`, `ASCENDING`, `BY`, `DESC`, `DESCENDING` and `ON` do not fall naturally under "subclauses".

A far more intuitive place would be under a new "Modifiers" section (as these modify some other keyword - most notably, a clause)